### PR TITLE
Added Product listing page with filters

### PR DESCRIPTION
**Pull request by TusharJade**
### Added product listing page with filter
Screenshot (35)

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Good to go

**Review by sumitch29 — APPROVED** [review]
Looks good to me

File: HTML and CSS/Product listing page with filters/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-family: 'Nunito', sans-serif;
+}
+
+body{
+    background-color: #f1f3f6;
+}
+
+.sort-byname {
```
Comment (sashaboi) [inline HTML and CSS/Product listing page with filters/product-listing.css:21]: A hover property to this element changing the cursor to pointer and bg color a little , might make it look more interractive? Hope this helps, 
Onkar Deshpande

File: HTML and CSS/Product listing page with filters/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+.sortby-container {
+    border: 1px solid gainsboro;
+    box-shadow: 0.1px 0.1px 0.2px 0.6px gainsboro;
+    width: max-content;
+    padding: 0.5rem 1rem;
+    background-color: white;
+}
+
+.all-contain {
```
Comment (sashaboi) [inline HTML and CSS/Product listing page with filters/product-listing.css:47]: A hover element here might help as well , and to the specific checkboxes as well maybe to make it look more interractive maybe ? 
Onkar Deshpande

**Review by sashaboi — COMMENTED** [review]
Overall the layout looks amazing , looking forward to further updates !

**Pull request merged**